modules/navier_stokes/doc/intro_slides/slides/index.md

```diff
@@ -0,0 +1,227 @@
+# The Navier-Stokes Module
+
+!style halign=center fontsize=120%
+!datetime today format=%B %Y
+
+!---
+
+# Overview
+
+Main capabilities:
+
+- Incompressible fluid flows
+- Weakly-compressible fluid flows
+- Compressible fluid flows
+- Free flows and flows in porous media describing homogenized structures
+
+Utilizies the following techniques:
+
+- Continuous Finite Element Discretization (maintained, but not intensely developed)
+- Finite Volume Discretization (current development direction)
+
+!---
+
+# The Navier-Stokes equations ([!citet](radman2021development))
+  style=font-size:26pt
+
+!style! fontsize=80%
+
+- Conservation of +mass+:
+
+!equation id=mass-eq
+\frac{\partial \gamma \rho}{\partial t} + \nabla \cdot \left( \rho \vec{u} \right)  = 0
+
+- Conservation of (linear) +momentum+:
+
+!equation id=momentum-eq
+\frac{\partial \rho  \vec{u}}{\partial t} + \nabla \cdot \left(\gamma^{-1}  \rho \vec{u} \otimes \vec{u}\right)
+= \nabla \cdot \left(\gamma \mu_\text{eff} \left(\nabla\vec{u}_I + \left(\nabla \vec{u}_I\right)^T-\frac{2}{3}\nabla\cdot\vec{u}_I\mathbb{I}\right)\right) -\gamma\nabla p + \gamma  \rho \vec{g} -  W  \vec{u}_I
+
+!style-end!
+
+!row!
+!col! width=30%
+
+!style! fontsize=70%
+- $\gamma$ - porosity
+- $\vec{u}_I$ - real (or interstitial) velocity
+- $\vec{u} \equiv \gamma \vec{u}_I$ - superficial velocity
+!style-end!
+
+!col-end!
+
+!col! width=60%
+
+!style! fontsize=70%
+- $\rho = \rho(p,T)$ - density
+- $\mu_{eff}$ - effective dynamic viscosity (laminar + turbulent)
+- $p$ - pressure
+- $W$ - friction tensor (using correlations)
+!style-end!
+
+!col-end!
+!row-end!
+
+!style! fontsize=80%
+- If $\gamma=1$ then $\vec{u}=\vec{u}_I$ and we get the original Navier-Stokes equations back.
+!style-end!
+
+
+!---
+
+# The Navier-Stokes equations (contd.)
+  style=font-size:26pt
+
+- These are supplemented by the conservarion of +energy+:
+
+  +Assumptions:+ Weakly-compressible fluid, kinteic energy of the fluid can be neglected
+
+!equation id=enthalpy-eq
+\frac{\partial \gamma \rho c_p T}{\partial t} + \nabla \cdot \left(  \rho c_p T \vec{u}\right)
+= \nabla \cdot \left(\kappa_f \nabla T \right) - \alpha (T - T_s) + S_{ext}
```

Comment:
```suggestion
\frac{\partial \gamma \rho c_p T}{\partial t} + \nabla \cdot \left(  \rho c_p T \vec{u}\right)
= \nabla \cdot \left(\gamma \kappa_f \nabla T \right) - \gamma \alpha (T - T_s) + \gamma S_{ext}
```

Reply:
The above is the mathematically correct formulation. Feel free to disregard if this is implemented differently in the code. I didn't check

Reply:
for diffusivity it depends how you chose to factor in the effect of porosity. Both options are available in the kernel. 

For the S_ext term we dont have it in any simulation right now, it's conventions as well imo

For the alpha dT term we should discuss.

Reply:
I assumed gamma is in kappa_s, I followed the description from the M2 report. I'll leave it like that for now, we can change it later.

Reply:
Actually, I don't think we have gamma on alpha*dT in this formulation. I assumed it is some sort of alpha = alpha_hx * a_v where a_v is a vlumetric surface m2/m3.

Reply:
Also, I did not define what S_ext is, you can do whatever you want with it. I just added an example source here. In some cases the source term is already compensated with a volume fraction in other apps (I think this was the issue with Cole's input, no?).